chroma-manager/tests/unit/lib/iml_unit_test_case.py

```diff
@@ -2,7 +2,10 @@
 
 from chroma_core.models import Command
 from chroma_core.services.log import log_register
+from chroma_common.lib import util
 
+# Manager unit tests should not use threading, see comment in util.py
+util._use_threads_default = False
```

Comment:
yes because we have split out AgentUnitTestCase from ImlUnitTestCase and we don't want tests inheriting from the latter using threads by default, whereas we do for the former